groups-rule-mapping.md

```diff
@@ -0,0 +1,118 @@
+# External to local groups mapping 
```

Comment:
this file is present in `elasticsearch-details` folder

Reply:
Oh shoot! Missed this.